src/main_service/main_service.rs

```diff
@@ -1,48 +1,101 @@
-use std::{io::{Error, ErrorKind}, net::{TcpStream, ToSocketAddrs, UdpSocket}, os::unix::net::SocketAddr, sync::{atomic::{AtomicBool, Ordering}, mpsc::Sender, Arc, Mutex}, thread, time::Duration};
+use std::{io::{Error}, net::{ToSocketAddrs, UdpSocket}, os::unix::net::SocketAddr, sync::{atomic::{AtomicBool, Ordering}, mpsc::Sender, Arc, Mutex}, thread, time::Duration};
 use log::{info, warn};
-use sal_sync::services::{conf::conf_tree::ConfTree, entity::{
+use sal_sync::services::{conf::conf_tree::{self, ConfTree}, entity::{
         name::Name, object::Object, point::point::Point
         // services::{
         //     services::Services,
         //     service::service::Service,
         //     service::service_handles::ServiceHandles, 
         // }, 
 }, service::{service::Service, service_cycle::ServiceCycle, service_handles::ServiceHandles}, types::type_of::TypeOf};
-
-use crate::amplitude;
-
+use serde::de::value;
 use super::main_service_config::MainServiceConf;
-//
-//
-// impl ServiceConfig{
-//     //
-//     ///
-//     pub fn new(conf: &serde_yaml::Value) -> Self{
-//         let ampl = conf["ample"].as_i64().unwrap_or(0.0) as f64;
-//         let phi = conf["phi"].as_i64().unwrap_or(0.0) as f64;
-//         Self{
-//             ampl,
-//             phi,
-//         }
-//     }
-// }
-//
-//
+use crate::Buffer;
+///
+/// Struct `UpdHeader`
+/// - `syn` - message starts with
+/// - `addr` - address of the input channel
+/// - `type` - type of values in the array in data field in struct `UpdMessage`
+/// - `count` - length of the array in the data field in struct `UpdMessage`
 pub struct UdpHeader {
     pub syn: u8,
     pub addr: u8,
     pub r#type: u8,
     pub count: u8,
-    pub data: [u8; 1024],
 }
+//
+//
+impl UdpHeader{
```

Comment:
Move to the `udp_header.rs` file
- develop in the Issie-PR pair
- may bee unit test ?